modules/web/src/app/node-data/basic/provider/nutanix/component.ts

```diff
@@ -102,6 +108,8 @@ export class NutanixBasicNodeDataComponent extends BaseFormValidator implements
   selectedSubnet = '';
   subnetLabel = SubnetState.Empty;
   categories: NutanixCategory[] = [];
+  initialCategories: Category[];
```

Comment:
Just to differentiate it more from `categories`
```suggestion
  initialSelectedCategories: Category[];
```